.github/workflows/test-phpunit.yml

```diff
@@ -121,17 +123,18 @@ jobs:
 
       - name: Install dependencies
         run: |
-          composer update --ansi --no-interaction
-          composer remove --ansi --dev --unused -W rector/rector phpstan/phpstan friendsofphp/php-cs-fixer nexusphp/cs-config
+          composer update --ansi --no-interaction ${{ matrix.composer-flag }}
+          composer remove --ansi --dev --unused -W ${{ matrix.composer-flag }} -- rector/rector phpstan/phpstan friendsofphp/php-cs-fixer nexusphp/cs-config
           php -r 'file_put_contents("vendor/laminas/laminas-zendframework-bridge/src/autoload.php", "");'
         env:
           COMPOSER_AUTH: ${{ secrets.COMPOSER_AUTH }}
 
-      - name: Profile slow tests in PHP 7.4 MySQLi for now
-        if: matrix.php-versions == '7.4' && matrix.db-platforms == 'MySQLi'
+      - name: Profile slow tests in PHP 8.0 MySQLi for now
+        if: matrix.php-versions == '8.0' && matrix.db-platforms == 'MySQLi'
```

Comment:
This is a change I would like to see now. If the rest of the 8.1 workflow takes a while please consider lifting this out.

Reply:
You mean you want to see profiling annotations for each matrix of 7.3 to 8.0 and MySqli to Sqlsrv? GHA allows a limit of 10 warnings per matrix so that would be 160 annotations run.

Reply:
No I meant that all single-version tests be run on the latest stable PHP. I think it is moot now that we're close to the next release.